src/main/resources/modules/uti/ed_path.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "UTI_ED",
```

Super minor but I was confused for a second when I came back to this module since the name in the UI didn't match the filename and I wasn't sure which "UTI ED" module I was looking at. (I do think prefixing the submodule with the directory name is nice for clarity too even though technically it also doesn't match the filename)
```suggestion
  "name": "UTI_ED_Path",
```